lib/hyrax/hirmeos/metrics_tracker.rb

```diff
@@ -15,7 +15,27 @@ def submit_to_hirmeos(work)
     client.post_work(resource_to_hirmeos_json(work))
   end
 
+  def submit_files_to_hirmeos(file_set)
```

Comment:
Since this is taking a single file set now how about renaming this to `submit_file_to_hirmeos`